AUTHORS

```diff
@@ -70,6 +70,7 @@ Dave Du Cros <davidducros@gmail.com>
 Dave Henderson <dhenderson@gmail.com>
 David Deng <daviddengcn@gmail.com>
 David Jannotta <djannotta@gmail.com>
+David Ji <github.com/davidji99>
```

Comment:
Sorry, please keep the AUTHORS file consistent with email addresses only.

Reply:
Oh okay @gmlewis. I only did my github user page since I saw a few references of it:
- https://github.com/google/go-github/blob/master/AUTHORS#L25
- https://github.com/google/go-github/blob/master/AUTHORS#L63

Reply:
Yikes.  OK. I guess I let those slip by.  Feel free to go back then to your GitHub user page. Sorry for the noise.

Reply:
Oh okay. No problem.